AK/Platform.h

```diff
@@ -27,6 +28,16 @@
 #    define AK_OS_BSD_GENERIC
 #endif
 
+#if defined(__CYGWIN__) || defined(_WIN32)
+#    define AK_OS_WIN32
+#    undef __POSIX_VISIBLE
+#    define __POSIX_VISIBLE 200809
```

Comment:
I think that eventually, on windows, we should not rely on POSIX. This seems like a thing that is going to be hard find when needed. Please ad a compile time warning.

Reply:
No such thing as `warnings` in serenity, they're all treated as errors